src/components/Register/Register.test.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { unwrappedRegister as Register } from '../Register/Register';
+
+const props = {
+  auth: {
+    isLoading: true,
+  },
+  signupUser: jest.fn(),
+};
+const shallowWrapper = shallow(<Register {...props} />);
```

Comment:
We can use only mount here instead of having both mount and shallow.

Reply:
I have removed shallow now and used mount test-wide